src/applications/vaos/new-appointment/newAppointmentFlow.js

```diff
@@ -358,162 +158,242 @@ const flow = {
+      },
     },
     contactInfo: {
-      ...flow.contactInfo,
-      url: featureBreadcrumbUrlUpdate
-        ? 'contact-information'
-        : '/new-appointment/contact-info',
+      url: 'contact-information',
+      label: 'How should we contact you?',
+      next: 'review',
+    },
+    home: {
+      url: '/',
     },
     preferredDate: {
-      ...flow.preferredDate,
-      url: featureBreadcrumbUrlUpdate
-        ? 'preferred-date'
-        : '/new-appointment/preferred-date',
+      url: 'preferred-date',
+      label: 'When are you available for this appointment?',
+      next: 'selectDateTime',
     },
     reasonForAppointment: {
-      ...flow.reasonForAppointment,
-      url: featureBreadcrumbUrlUpdate
-        ? 'reason'
-        : '/new-appointment/reason-appointment',
+      url: 'reason',
+      label: 'What’s the reason for this appointment?',
+      next(state) {
+        if (
+          isCCFacility(state) ||
+          getNewAppointment(state).flowType === FLOW_TYPES.DIRECT
+        ) {
+          return 'contactInfo';
+        }
+
+        return 'visitType';
+      },
     },
     requestDateTime: {
       ...flow.requestDateTime,
-      url: featureBreadcrumbUrlUpdate
-        ? 'va-request/'
-        : '/new-appointment/request-date',
-    },
-    ccRequestDateTime: {
-      ...flow.requestDateTime,
-      url: featureBreadcrumbUrlUpdate
-        ? 'community-request/'
-        : '/new-appointment/request-date',
-    },
-    root: {
-      url: featureBreadcrumbUrlUpdate
-        ? '/my-health/appointments'
-        : '/health-care/schedule-view-va-appointments/appointments/',
+      url: 'va-request/',
     },
     review: {
-      ...flow.review,
       label:
         FLOW_TYPES.DIRECT === flowType
           ? 'Review and confirm your appointment details'
           : 'Review and submit your request',
-      url: featureBreadcrumbUrlUpdate ? 'review' : '/new-appointment/review',
+      url: 'review',
+    },
+    root: {
+      url: '/my-health/appointments',
     },
     scheduleCerner: {
-      ...flow.scheduleCerner,
-      url: featureBreadcrumbUrlUpdate
-        ? 'how-to-schedule'
-        : '/new-appointment/how-to-schedule',
+      url: 'how-to-schedule',
+      label: 'How to schedule',
     },
     selectDateTime: {
-      ...flow.selectDateTime,
-      url: featureBreadcrumbUrlUpdate
-        ? 'date-time'
-        : '/new-appointment/select-date',
+      url: 'date-time',
+      label: 'What date and time do you want for this appointment?',
+      next: 'reasonForAppointment',
+    },
+    selectProvider: {
+      url: 'provider',
+      label: 'Which provider do you want to schedule with?',
+      next: 'preferredDate',
+    },
+    typeOfAppointment: {
+      url: '/new-appointment',
+      // Temporary stub for typeOfAppointment which will eventually be first step
+      // Next will direct to type of care or provider once both flows are complete
+      next: 'typeOfFacility',
```

Comment:
`typeOfAppointment` can be deleted since we moved away from using the URL "/new-appointment"